core/src/main/java/org/kiwix/kiwixmobile/core/dao/NewBookmarksDao.kt

```diff
@@ -40,6 +40,8 @@ class NewBookmarksDao @Inject constructor(val box: Box<BookmarkEntity>) {
   }.find()
     .map(::BookmarkItem)
 
+  fun bookmarks() = box.asFlowable().map { it.map(::BookmarkItem) }
```

Comment:
I imagine some corresponding thing can be deleted with the addition of this method?